### DISCUSSION TITLE
Use profiles to change settings for compiling?

### DISCUSSION THREAD

**Discussion by bvstone** [codefori/vscode-ibmi#1248] [Q&A]
Using Code for IBM i v1.8.0.

I have a case where when working on certain projects I want to change, at the very least, the TGTRLS for the compile commands.  I tried using profiles in VSC but that didn't seem to do anything.  I couldn't find where to change it except in the settings.json file, but I couldn't get it to stick.

Just wondering if there's a way to do this.

When I say profiles, I mean in VSC the gear icon in the lower left.

Also, when changing between profiles the view of my system, folders, source members, etc goes blank.  I have to restart VSC to get them back.

Thanks!

**Comment by worksofliam** [comment]
VS Code profiles won't work for what you're looking to do. You really want Code for IBM i profiles, which came out before the Profiles feature in VS Code, and now unfortunately share a name.

https://halcyon-tech.github.io/docs/#/pages/settings/profiles
